pyaedt/generic/general_methods.py

```diff
@@ -343,4 +343,8 @@ def time_fn( fn, *args, **kwargs ):
     return results
 
 def isclose(a, b, rel_tol=1e-9, abs_tol=0.0):
-    return abs(a-b) <= max( rel_tol * max(abs(a), abs(b)), abs_tol )
\ No newline at end of file
+    return abs(a-b) <= max( rel_tol * max(abs(a), abs(b)), abs_tol )
+
+
+def is_number(a):
+    return str(a).replace(".", "").replace("+", "").replace("-", "").replace("e","").replace("E","").isnumeric()
```

Comment:
```suggestion
    return str(a).replace(".", "").replace("+", "").replace("-", "").replace("e", "").replace("E", "").isnumeric()
```

Reply:
Is there any reason why `eval` wouldn't work here?  Something like:
```py
try:
    float(eval(a))
    return True
except:
    return False

Reply:
I would tend to agree with Alex here.

Reply:
changed